tools/cli/pkg/command/upgrade_cluster.go

```diff
@@ -0,0 +1,60 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/common/orchestration"
+	"github.com/kyma-project/control-plane/tools/cli/pkg/logger"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+type UpgradeClusterCommand struct {
+	UpgradeCommand
+	cobraCmd *cobra.Command
+}
+
+func NewUpgradeClusterCommand() *cobra.Command {
+	cmd := UpgradeClusterCommand{UpgradeCommand: UpgradeCommand{}}
+	cobraCmd := &cobra.Command{
+		Use:   "cluster --target {TARGET SPEC} ... [--target-exclude {TARGET SPEC} ...]",
+		Short: "Upgrades Kubernetes cluster on one or more Kyma Runtimes.",
+		Long: `Upgrade Kubernetes cluster on targets of Runtimes.
```

Comment:
In the long description, mention that Kubernetes version and/or machine image is being upgraded, e.g. "Upgrade Kubernetes version and/or machine image on targets of Runtimes"